core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -690,6 +690,42 @@ public enum CoreError implements ScalarDbError {
       ""),
   DATA_LOADER_ERROR_METHOD_NULL_ARGUMENT(
       Category.USER_ERROR, "0151", "Method null argument not allowed", "", ""),
+  OUT_OF_RANGE_COLUMN_VALUE_FOR_DATE(
+      Category.USER_ERROR,
+      "0152",
+      "This Date column value is out of the valid range. It must be from 1000-01-01 through 9999-12-12. Value: %s",
+      "",
+      ""),
+  SUBMICROSECOND_PRECISION_NOT_SUPPORTED_FOR_TIME(
+      Category.USER_ERROR,
+      "0153",
+      "This Time column value precision cannot be smaller than a microsecond. Value: %s",
```

Comment:
Just to be specific, I think we should change `a` to `one`.

```suggestion
      "This Time column value precision cannot be smaller than one microsecond. Value: %s",
```